frontend/src/app/[locale]/subscribe/actions.ts

```diff
@@ -0,0 +1,134 @@
+"use server";
+
+import { environment } from "src/constants/environments";
+import { TFn } from "src/types/intl";
+import { z } from "zod";
+
+import { getTranslations } from "next-intl/server";
+import { redirect } from "next/navigation";
+
+import { ValidationErrors } from "src/components/subscribe/SubscriptionForm";
+
+type sendyResponse = {
+  validationErrors: ValidationErrors;
+  errorMessage: string;
+};
+
+export async function subscribeEmail(
+  prevState: sendyResponse,
+  formData: FormData,
+) {
+  const t = await getTranslations();
```

Comment:
if we do 
```suggestion
  const t = await getTranslations("Subscribe");
```
do we only close over that part of the messaging object? if so that could be a nice middle ground here

Reply:
`await getTranslations("Subscribe");` caused a linting error.